### PR TITLE
DEVDOCS-5911 [new]: MSF/Stencil, add storefront selector guide

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-5911]

## What changed?

- Add guide for MSF Storefront selector 

## Release notes draft

We are happy to announce the newly-released storefront selector feature in Stencil themes. Shoppers can now choose which storefront they want to browse when they visit your storefront.  

[DEVDOCS-5911]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5911?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by bc-yevhenii-buliuk** [comment]
@bc-andreadao @BC-krasnoshapka how do you think should we warn about wrong bundle change options?

**Review by bc-yevhenii-buliuk — DISMISSED** [review]

**Review by BC-krasnoshapka — APPROVED** [review]
LGTM! Just noticed few links that are rendered as markdown.

File: docs/stencil-docs/localization/msf-storefront-selector.mdx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+```
+
+After a successful build, a zip file will appear in the project root with the theme name and version, for example `Cornerstone-6.14.0.zip`.
+
+
+## Step 6: Upload the bundle to the control panel
+
+To [upload the bundle](https://support.bigcommerce.com/s/article/Stencil-Themes?language=en_US#download-upload) with the updated selector, go to Themes in the control panel and click the Upload Theme button. Select the bundle created in the previous step and upload it.
+
+Apply the bundle and open your store. If the selector is not displayed, go to [Page Builder](https://support.bigcommerce.com/s/article/Page-Builder) and navigate to **Theme Styles > Header and Footer > Utility navigation > Show storefront selector** and check this checkbox. Save the changes and refresh the store.
```
Comment (bc-traciporter) [inline docs/stencil-docs/localization/msf-storefront-selector.mdx:125]:
```suggestion
Apply the bundle and open your store. If the selector is not displayed, go to [Page Builder](https://support.bigcommerce.com/s/article/Page-Builder) and navigate to **Theme Styles** > **Header and Footer** > **Utility navigation** > **Show storefront selector** and check this checkbox. Save the changes and refresh the store.
```

File: docs/stencil-docs/localization/msf-storefront-selector.mdx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+To clone the Cornerstone repository, run the command below in the terminal of your operating system:
+
+```bash
+git clone https://github.com/bigcommerce/cornerstone.git
+```
+
+## Step 2: Install Stencil CLI
+
+Open the cloned repository in any code editor (IDE). Open your IDE-terminal, which is where you can run the rest of the steps. Depending on the operating system you are using, [install the Stencil CLI](/docs/storefront/stencil/cli/install) that should be supported by the appropriate version of Node.js.
```
Comment (bc-traciporter) [inline docs/stencil-docs/localization/msf-storefront-selector.mdx:40]:
```suggestion
Open the cloned repository in any code editor (IDE). Open your IDE-terminal, which is where you can run the rest of the steps. Depending on the operating system, [install the Stencil CLI](/docs/storefront/stencil/cli/install) supported by the appropriate version of Node.js.
```

**Review by bc-traciporter — COMMENTED** [review]
Left comments for you to review

**Review by bc-traciporter — APPROVED** [review]
Approved

**Pull request merged**